src/controllers/ContactController.php

```diff
@@ -55,24 +89,17 @@ public function contact(Request $request, PDO $db)
             throw new Exception($message, 400);
         }
 
-        // run it by akismet if we have it
-        if (isset($this->config['akismet']['apiKey'], $this->config['akismet']['blog'])) {
-            $spamCheckService = new SpamCheckService(
-                $this->config['akismet']['apiKey'],
-                $this->config['akismet']['blog']
-            );
-            $isValid          = $spamCheckService->isCommentAcceptable(
-                $data['comment'],
-                $request->getClientIP(),
-                $request->getClientUserAgent()
-            );
-            if (!$isValid) {
-                throw new Exception("Comment failed spam check", 400);
-            }
+        $isValid = $this->spamCheckService->isCommentAcceptable(
+            $data,
+            $request->getClientIP(),
+            $request->getClientUserAgent()
+        );
+
+        if (!$isValid) {
```

Comment:
Take it or leave it, but we don't need the temp var here.

Reply:
and done